demo/README.md

```diff
@@ -38,7 +38,7 @@ The hashes for the default public keys present in iot-platform-sdk and supply-ch
 | 834F83875910C8507CE935BE2F947DCF854E6554C3ACB79893ACF91220EA5D8B | IOT Platform SDK - ECDSA 384 |
 | B4E95FB7062303BEB84FBB606ED75CCE99D1C4B6CC88F71E65286CAD7C74F3A5 | IOT Platform SDK - RSA 2048 |
 
-To whitelist other credentials, see instructions in the [Secure Device Onboard Rendezvous Service README](https://github.com/secure-device-onboard/rendezvous-service/#trust-management).
+To allowlist other credentials, see instructions in the [Secure Device Onboard Rendezvous Service README](https://github.com/secure-device-onboard/rendezvous-service/#trust-management).
```

Comment:
Same as above.

'To add other credentials in allowlist'

Reply:
Done